framework/csrf/handlers.py

```diff
@@ -23,7 +31,7 @@ def before_request():
     # Reuse token if already set
     csrf_token = _get_token()
     if not csrf_token or g.get('csrf_cookie_needs_reset', False):
-        csrf_token = _get_new_csrf_token()
+        csrf_token = _mask_cipher_secret(_get_new_csrf_string())
```

Comment:
Just curious, why we have to do this now after the upgrade?

Reply:
Because in django 4.0 removed `_get_new_csrf_token` has been removed

Reply:
oh, I mis-read this `_get_new_csrf_string()` as `_get_new_csrf_token()`